### PR TITLE
Adding defense against run time exceptions of dict random. Making the…

### PR DESCRIPTION
**Pull request by Khalian**
… function return an optional.

## Checklist

- [x] New Extension
- [x] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by Khalian** [comment]
To address the issue given in https://github.com/goktugyil/EZSwiftExtensions/issues/413

File: Sources/DictionaryExtensions.swift [hunk 1]
```diff
@@ -8,10 +8,10 @@
 import UIKit
 
 extension Dictionary {
+   
     /// EZSE: Returns the value of a random Key-Value pair from the Dictionary
-    public func random() -> Value {
-        let index: Int = Int(arc4random_uniform(UInt32(self.count)))
-        return Array(self.values)[index]
+    public func random() -> Value? {
+        return Array(self.values).random()
```
Comment (dfrib) [inline Sources/DictionaryExtensions.swift]: (`self` could be omitted here, if preferred: `return Array(values).random()`)

Comment (Khalian) [inline Sources/DictionaryExtensions.swift]: I presume idiomatic swift uses implicits over explicits?

Comment (dfrib) [inline Sources/DictionaryExtensions.swift]: Yes I believe so; I generally only spot common `self` usage in the stdlib's source code when subscripting (in which case in naturally must be present; `self[subscript]`), but at the same time it seems they (stdlib) don't follow this implicitness extremely strictly (I've seen both, but implicits are more frequent).

Comment (Khalian) [inline Sources/DictionaryExtensions.swift]: Done

**Comment by Khalian** [comment]
I am merging this change, since random is susceptible to runtime exceptions otherwise.

**Pull request merged**